.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+### Current Behavior
+
+- What code are you running and what is happening?
+- Include a screenshot if it makes sense.
+
+### Expected Behavior
+
+- What do you expect should be happening?
+- Include a screenshot if it makes sense.
```

Comment:
I'd not make "adding a screenshot" a separate item since it's not necessary in >90% of reports.

Reply:
"Remove this here".
You mean remove the last line?

'how to make screenshot of expected behaviour'.
That depends entirely on the issue therefore the "if it makes sense". 
An example could be; `build ... test ....` might give some output where you expect the output to equal that of `testinvoke`. This probably it still doable in words but there might be cases where you create a screenshot and draw an error to illustrate a problem+solution. Also I'm going to assume that the future build in debugger will be slightly more visual and then this can apply more often.